### DISCUSSION TITLE
Reproduction rate stopped on 2022-01-03

### DISCUSSION THREAD

**Discussion by udra69** [owid/covid-19-data#2254] [Q&A] [answered]
I could not find anything about stopping calculating R. Can you tell me why this happened?
thx, Ulli

**Comment by lucasrodes** [comment, marked as answer]
Hi @udra69,
Thanks for bringing this to our attention!

We currently source the reproduction rate R from project https://github.com/crondonm/TrackingR, in particular [this file](https://github.com/crondonm/TrackingR/blob/main/Estimates-Database/database.csv).

Based on the authors' guidelines, this data can have up to three days delay (now it is ~7days). I have opened issue crondonm/TrackingR#5 to track this.
